mozilla/blurts-server
# Email design changes requested by Sally

# References: 
Jira: MNTOR-3088
Figma: https://www.figma.com/file/CaEKIhvSJqf6KNIMzSkt40/Concepts-for-Monitor-MVP-Redesign?type=design&node-id=10719%3A13997&mode=design&t=Z7sQgJekpuuOmV9z-1




# Description

Sally thought content was pushed too much below the fold, so Zachery removed the header.  He also aligned the illustrations with the ones on the fixed tab.

# Screenshot (if applicable)

![image](https://github.com/mozilla/blurts-server/assets/4251/3604cb03-9c38-4b2a-a2b8-7038e5178e09)

![image](https://github.com/mozilla/blurts-server/assets/4251/83d7fce7-13ca-4fab-9413-9c7b9706d86c)

![image](https://github.com/mozilla/blurts-server/assets/4251/5c2679d3-11d2-4f5a-bcae-34af5030d205)

# How to test

Send the emails to yourself via `/admin/emails`, or view them in Storybook (but make sure you also have the regular server running, as that serves up the images).

# Checklist (Definition of Done)
- [ ] I've added a unit test to test for potential regressions of this bug. - N/A, no logic changes
